fuel-txpool/src/service.rs

```diff
@@ -179,11 +181,9 @@ impl Context {
 
                     tokio::spawn( async move {
                         let txpool = txpool.as_ref();
-                        match new_transaction.unwrap() {
-                            TransactionBroadcast::NewTransaction ( tx ) => {
-                                let txs = vec!(Arc::new(tx));
-                                TxPool::insert(txpool, db.as_ref().as_ref(), tx_status_sender, txs).await
-                            }
+                        if let GossipData { data: Some(TransactionBroadcast::NewTransaction ( tx )), .. } =  new_transaction.unwrap() {
+                            let txs = vec!(Arc::new(tx));
+                            TxPool::insert(txpool, db.as_ref().as_ref(), tx_status_sender, txs).await;
```

Comment:
Should we implement the message validation response here? Or add a todo for this later?

Reply:
I will create an issue for it

Reply:
https://github.com/FuelLabs/fuel-core/issues/704